manifests/syslog_loggers.pp

```diff
@@ -0,0 +1,122 @@
+define vcenter::syslog_loggers (
+  Integer          $esxi_version             = undef,
+  String           $transport_string         = undef,
+  Hash             $logger_options           = {}
+) {
+  #Common loggers available in a ESXi machine 
+  #Options are mentioned here for reference only
+  #please uncomment the ones you are interested in 
```

Comment:
@tuxfight3r We should not be encouraging users to make modifications to module code. May I suggest that you change this to something like ....

```puppet
$common = $::vcenter::logger_defaults
```

An `init.pp` base class (which currently doesn't exist) as:

```puppet
class vcenter (
  Hash  $logger_defaults = {}
) {
}
```

These options should *definately* be documented, but not in a way that encourages (and indeed tells) the user to edit the module to enable them - they should be configured in Hiera.